dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -72,3 +72,22 @@ namespaces:
         tags: [channel-name]
         type: meter
         docs: Marked every time we reshuffle all the nodes.
+
+  dialogue.roundrobin:
+    docs: Instrumentation for the ROUND_ROBIN node selection strategy (currently implemented by BalancedChannel).
+    metrics:
+      success:
+        type: meter
+        tags: [channel-name, hostIndex]
+        docs: Meter of the requests that were successfully made, tagged by the index of the host. (Note if there are >10
+          nodes this metric will not be recorded).
+
+  dialogue.balanced:
```

Comment:
Thoughts on including score in the name for clarity?

Reply:
yep the metric comes out as `dialogue.balanced.score`, this is just the namespace

Reply:
Oh geez... I can't read.